src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java

```diff
@@ -74,6 +77,7 @@ public Command(JDAImpl api, Guild guild, DataObject json)
         this.options = parseOptions(json, OPTION_TEST, Option::new);
         this.groups = parseOptions(json, GROUP_TEST, SubcommandGroup::new);
         this.subcommands = parseOptions(json, SUBCOMMAND_TEST, Subcommand::new);
+        this.version = json.getUnsignedLong("version");
```

Comment:
Just in case, lets use the id as a fallback:

```suggestion
        this.version = json.getUnsignedLong("version", id);
```

Reply:
Too late